src/extras/webgl/TilingSpriteRenderer.js

```diff
@@ -94,9 +75,41 @@ export class TilingSpriteRenderer extends core.ObjectRenderer {
             }
         }
 
-        const shader = isSimple ? this.simpleShader : this.shader;
+        return isSimple;
+    }
 
-        renderer.bindShader(shader);
+    /**
+     * For extra pixi plugins use
+     *
+     * @param {PIXI.TilingSprite} ts tiling sprite
+     * @param {boolean} isSimple is it repeating texture or not
+     * @param {PIXI.Shader} shader, must be set accordingly to isSimple
+     */
+    renderWithShader(ts, isSimple, shader)
```

Comment:
Actually, we cant just set ANY shader to TilingSprite and Sprite. What if we make some container for uniforms too, like we do with filter?